Tasks/Assignment 3/hero.js

```diff
@@ -0,0 +1,8 @@
+class Hero {
```

Comment:
it was asked to create the constructor function not the class
file name hero1.js is really bad please change it